src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -474,6 +474,41 @@ sourceFileAzure, getSourceStorageAccountPrimarySharedKey()))
     DatasetSpecificationModel datasetSchema = datasetModel.getSchema();
 
     // dataset ingest
+    // Ingest Metadata - 1 row in ARRAY mode
```

Comment:
Could this new code be written as separate method called from here? This method is already really long and it looks like the only argument that needs to be passed in is `tableRowCount`.

Reply:
Yeah agreed, it might be more readable to pull it into its own method